composer.json

```diff
@@ -29,22 +29,26 @@
         "symfony-cmf/core-bundle": "^1.1|^2.0",
         "matthiasnoback/symfony-dependency-injection-test": "^0.7.4",
         "matthiasnoback/symfony-config-test": "^1.3.1",
+        "symfony/phpunit-bridge": "^3.2",
         "burgov/key-value-form-bundle": "^1.0"
     },
-    "minimum-stability": "dev",
```

Comment:
i think with this things are not installable anymore on travis, until we have released things